### PR TITLE
Feature/arci notebook

### PR DESCRIPTION
**Pull request by lixinsu**
#404  
link to the notebook, click [here](https://github.com/NTMC-Community/MatchZoo/blob/feature/arci_notebook/notebooks/wikiqa/arci.ipynb)

### CODE REVIEW

File: notebooks/wikiqa/arci.ipynb [hunk 1]
```diff
@@ -0,0 +1,749 @@
...
+   "source": [
+    "#### Preproceesing with pre-trained embeddings\n",
+    "\n",
+    "We run the processor in three lines of code.  \n",
+    "If we use **the randomly initialized embeddings**, just keep default value for *embedding_file*, and the preprocessing code is as follows:\n",
+    "```python\n",
+    "from matchzoo import preprocessor\n",
+    "arci_preprocessor = preprocessor.ArcIPreprocessor()\n",
+    "processed_tr = arci_preprocessor.fit_transform(train, stage='train')\n",
+    "processed_te = arci_preprocessor.fit_transform(test, stage='test')\n",
```
Comment (faneshion) [inline notebooks/wikiqa/arci.ipynb]: Here, the `state`  should be `evaluate` as it is to evaluate the performance of test set.
